prepare-image.sh

```diff
@@ -8,13 +8,15 @@ echo "install_weak_deps=False" >> /etc/dnf/dnf.conf
 # Tell RPM to skip installing documentation
 echo "tsflags=nodocs" >> /etc/dnf/dnf.conf
 
-dnf install -y python3 python3-requests 'dnf-command(config-manager)'
+dnf install -y python3 'dnf-command(config-manager)'
```

Comment:
nit but I guess we don't need the python3 here anymore given it's no longer run after the curl, and will get pulled in via deps from IRONIC_PKG_LIST?

Reply:
Howdy Steve o/, Updated the PR.